cluster/gce/gci/configure.sh

```diff
@@ -160,8 +160,15 @@ function validate-hash {
 # Get default service account credentials of the VM.
 GCE_METADATA_INTERNAL="http://metadata.google.internal/computeMetadata/v1/instance"
 function get-credentials {
+  local status=""
   curl --fail --retry 5 --retry-delay 3 --retry-connrefused --silent --show-error "${GCE_METADATA_INTERNAL}/service-accounts/default/token" -H "Metadata-Flavor: Google" -s | ${PYTHON} -c \
     'import sys; import json; print(json.loads(sys.stdin.read())["access_token"])'
+  status="$?"
+
+  if [[ "${status}" -ne 0 ]]; then
+    echo "Cannot download a default service account token for this VM." >&2
```

Comment:
not sure if we need `>&2`, looking at `configure.sh` most of the logging we have here is simply using echo and going to stdout.

Reply:
the problem with regular echo - it will be treated as the function output

Reply:
yep. this was the result of a confusing chain of responsibilities. Fixed